### PR TITLE
added Redux store

### PR DESCRIPTION
**Pull request by wulalasuperdope**
- run npm i
 - you should be able to see the App view

Changes:
- I added the redux folder, utils.js and store.js to the app

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -26,8 +26,9 @@ import {
 } from 'react-native/Libraries/NewAppScreen';
 
 import axios from 'axios';
-import {Button} from 'react-native-material-ui';
+import {Button, Card} from 'react-native-material-ui';
```
Comment (nries1) [inline App.js]: are we still using material ui?

File: ClimbingRoutes.js [hunk 2]
```diff
@@ -11,18 +11,35 @@ import {
...
+    routeList: [],
+  };
+
+  componentDidMount() {
+    axios
+      .get('https://climbar.herokuapp.com/api/climbingroutes')
+      .then(res => res.data)
+      .then(data => {
+        console.log(data);
+        this.setState({routeList: data});
```
Comment (nries1) [inline ClimbingRoutes.js]: I guess you're gonna switch this to redux? (Sorry if this is an old commit.)

File: App.js [hunk 3]
```diff
@@ -38,73 +38,124 @@ class App extends Component {
...
-              </Text>
-            </View>
-            <LearnMoreLinks />
-          </View>
-        </ScrollView>
-      </SafeAreaView>
-    </>
-  );
-};
+// const App: () => React$Node = () => {
```
Comment (nries1) [inline App.js]: can we delete?

File: App.js (hunk 1 above)
Comment (wulalasuperdope) [reply App.js]: I'm not. I leave it because I'm not sure if you want to keep the topnav.

File: ClimbingRoutes.js (hunk 2 above)
Comment (wulalasuperdope) [reply ClimbingRoutes.js]: Yep, working on it.

File: App.js (hunk 3 above)
Comment (wulalasuperdope) [reply App.js]: Sure

**Comment by nries1** [comment]
Thans @wulalasuperdope ! The code looks good but I haven't been able to run the app in this branch yet. Still getting a build error. I'm trying to resolve it now.

File: utils.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+export const getCookie = () =>
+  document.cookie.split(';').filter(c => /session_id=/.test(c))[0]
+    ? document.cookie
+        .split(';')
+        .filter(c => /session_id=/.test(c))[0]
+        .replace(/session_id=/, '')
+        .replace(/ /, '')
+    : null;
+
```
Comment (raymondng1) [inline utils.js]: don't think we would need this line of code anymore since this is not a web app anymore

Comment (wulalasuperdope) [reply utils.js]: Yes

**Pull request merged**